apps/site/lib/site_web/views/page_content_view.ex

```diff
@@ -110,6 +106,14 @@ defmodule SiteWeb.CMS.PageView do
     end
   end
 
+  @spec trim_and_downcase(String.t() | nil) :: String.t()
+  defp trim_and_downcase(s) do
+    case s do
+      nil -> nil
+      s -> String.downcase(String.trim(s))
+    end
+  end
+
   @spec alert_related?(MapSet.t(), Alerts.Alert.t()) :: boolean()
   defp alert_related?(project_paths, alert) do
     case get_mbta_url_path(alert.url) do
```

Comment:
this case actually doesn't seem to be doing anything unless I"m missing something, so along w/ the comment below, this could just get added to the chain 
```elixir
alert.uri
|> get_mbta_url_path()
|> get_project_url_paths()
...
```

Reply:
wait... did this change? I'm so confused, I swear lines 116-118 weren't there when reviewing this haha.

Reply:
![image](https://user-images.githubusercontent.com/526017/172425803-dedc0645-4f00-43b5-81b4-5eb8052073c7.png)
oh.